.github/workflows/ci.yml

```
@@ -0,0 +1,74 @@
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Set up Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v4
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: 'npm'
+          cache-dependency-path: web/packages/gateway/package-lock.json
+
+      - name: Cache Maven dependencies
+        uses: actions/cache@v3
```

actionlint (1.7.9)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)